src/modules/adhs/test.json

```diff
@@ -0,0 +1,51 @@
+{
+  "modules": [
+    "adhs"
+  ],
+  "setup": [
+    "mkdir -p /etc/azure-device-health-services/",
+    "touch /etc/azure-device-health-services/config.toml",
+    "cp /etc/azure-device-health-services/config.toml config.toml"
+  ],
+  "teardown": [
+    "cp config.toml /etc/azure-device-health-services/config.toml",
+    "rm -f config.toml"
+  ],
+  "steps": [
+    {
+      "type": "reported",
+      "component": "Adhs",
+      "object": "invalid",
+      "expect": {
+        "status": 22
+      }
+    },
+    {
+      "type": "desired",
+      "component": "Adhs",
+      "object": "desiredOptIn",
+      "payload": 1
+    },
+    {
+      "run": "grep -q 'Permission = \"Required\"' /etc/azure-device-health-services/config.toml"
+    },
```

Comment:
So you are proposing to replace the test recipes with these?

Why aren't the test recipes removed as well?

If we deviate from the agreed path, we need to have the changes proposed and discussed in team.